javalib/src/main/scala/java/net/URI.scala

```diff
@@ -800,23 +827,23 @@ object URI {
   }
 
   /** Upper-cases all URI escape sequences in `str`. Used for hashing */
-  private def normalizeEscapes(str: String): String = {
-    var i = 0
-    var res = ""
-    while (i < str.length) {
-      if (str.charAt(i) == '%') {
-        assert(str.length > i + 2, "Invalid escape in URI")
-        res += str.substring(i, i+3).toUpperCase()
-        i += 3
-      } else {
-        res += str.substring(i, i+1)
-        i += 1
+  private def normalizeEscapes(maybeStr: js.UndefOr[String]): js.UndefOr[String] =
```

Comment:
The body of this method should have `{}`.

Reply:
Fixed